65black/so-lite-react
# #162832547 Add Login Functionality

#### What does this PR do?
This PR adds the ability of a user to log into their account on StackO-lite.
This PR also updates the Navbar to change depending on whether a user is logged in or not.

#### Description of Task to be completed?
- add login functionality
- add tests for new feature
- add auth for public pages
- update signup page to use new auth
- update all tests affected by new changes

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Create an account #2 
- Navigate to `localhost:9000/login` and log in to your newly created account
- You should get redirected to the homepage on a successful login

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832547

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-26 at 8 34 20 pm" src="https://user-images.githubusercontent.com/36507972/50455339-ad87ca00-094d-11e9-80ca-ff5439be8229.png">
<img width="1440" alt="screenshot 2018-12-26 at 8 33 59 pm" src="https://user-images.githubusercontent.com/36507972/50455341-ad87ca00-094d-11e9-949d-3c5560f96df6.png">

#### Questions:
N/A
